packages/fxa-payments-server/src/store/sequences.ts

```diff
@@ -50,6 +51,13 @@ export const fetchCustomerAndSubscriptions =
     await Promise.all([dispatch(fetchCustomer())]).catch(handleThunkError);
   };
 
+export const fetchSubscriptionChangeEligibilityAndRefresh =
+  (plan: Plan) => async (dispatch: Function) => {
+    await Promise.all([
+      dispatch(fetchSubscriptionChangeEligibility(plan)),
+    ]).catch(handleThunkError);
```

Comment:
Q: Why wrap in `Promise.all([…])` if there is only one item/promise?

Reply:
Good catch! Removed. 🙂